### DISCUSSION TITLE
How to add a site title to the blog page?

### DISCUSSION THREAD

**Discussion by TheZoker** [facebook/docusaurus#5119] [Q&A] [answered]
How can I add a site title to the blog page? 

Ours just shows the main title:
![image](https://user-images.githubusercontent.com/1368405/124754404-79cc8880-df2a-11eb-8ef4-32fed3caa8a3.png)

While the docusaurus docs show `Blog | Docusaurus`
![image](https://user-images.githubusercontent.com/1368405/124754466-8ea91c00-df2a-11eb-84df-b5505c53e492.png)

I already tried `blogTitle: 'News'` but that did not help

**Comment by slorber** [comment, marked as answer]
Just tested this and `blogTitle: 'News'` works fine

Make sure to pass it as a plugin config option and not a themeConfig

https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-blog#configuration

**Reply by TheZoker** [reply]
Hmm strange. 
I made the change here: https://github.com/TheZoker/jsonforms2-website/blob/enhance-seo/docusaurus.config.js#L170

And here is the preview: https://60e5a28167201d00070a44f7--jsonforms-website.netlify.app/news

Any idea why this would not work?

**Reply by slorber** [reply]
> "@docusaurus/core": "2.0.0-alpha.73",

only possible reason I can think of

**Reply by slorber** [reply]
Inspected your PR here: https://github.com/eclipsesource/jsonforms2-website/pull/194

Nothing looks wrong to me. We previously had a few bugs regarding SEO titles but they should be fixed now

**Reply by TheZoker** [reply]
Yes I can confirm, that the old version is the issue here. Currently we need to hold back on the update, since one of the dependency is not yes ready for webpack 5. Thanks for the help!

**Reply by electrovir** [reply]
This is not working for me in version 3.5.

**Comment by manuelmeurer** [comment]
I have a similar problem - our blog shows the main `title` from the config as suffix in the meta title when displaying a blog post, but I'd like this to be the `blogTitle`.

Example:
https://uplink.tech/blog/2023/03/02/chance-statt-krise-the-power-of-freelancers/

Title is: Mit Freelancern die Chancen der Krise nutzen | Uplink Docs
Title should be: Mit Freelancern die Chancen der Krise nutzen | Uplink Blog

If this is not possible, can the "| Uplink Docs" suffix be removed completely?

Here's our config: https://github.com/uplinkhq/docs/blob/main/docusaurus.config.js

**Reply by Invidam** [reply]
same issue in 3.5.2
